Rocket.Chat/Models/File.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  File.swift
+//  Rocket.Chat
+//
+//  Created by Filipe Alvarenga on 11/04/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+class File: BaseModel {
```

Comment:
I imagine we can start using File in the future and remove the Attachment? 🤔

Reply:
@rafaelks it will depend on the API. I've created a different object for "File" because it seems that for the API they're two different things. The point is that many Attachments are basically files but not all Attachments can be represented by a File (text attachments, for example).

I think this is something that we will have to rethink together with the backend team.